packages/compass-indexes/src/components/search-index-template-dropdown/index.tsx

```diff
@@ -0,0 +1,74 @@
+import { ATLAS_SEARCH_TEMPLATES } from '@mongodb-js/mongodb-constants';
+import type { SearchTemplate } from '@mongodb-js/mongodb-constants';
+import React, { useState, useCallback } from 'react';
+import {
+  Select,
+  Option,
+  Icon,
+  css,
+  spacing,
+  Tooltip,
+} from '@mongodb-js/compass-components';
+
+const dropdownStyles = css({
+  display: 'flex',
+  overflow: 'hidden',
+  gap: 0,
```

Comment:
You don't need to specify gap if it's not needed / zero

Reply:
Yeah, I usually make it explicit even if it's not inherited. I do it even in CSS, but if it's a problem I can just remove it.

Reply:
Feel free to keep it, but specifying properties that seemingly do nothing just raises questions, especially after some time passed, for whoever will be working with this code or looking at it.

Reply:
I changed a bit the styles with your suggestions and removed the gap: 0 so it's now a bit simpler.